README.md

```diff
@@ -1,3 +1,20 @@
 # Typir
 
-A structure for generating validity checks associated with types.
\ No newline at end of file
+Engineering types for software languages in the web.
```

Comment:
Suggestion:

Typir is a framework for type systems and type checking in the web.